configs/config.json.example

```diff
@@ -169,7 +169,8 @@
           "enabled": true,
           "lure_attraction": true,
           "lure_max_distance": 2000,
-          "walker": "StepWalker"
+          "walker": "StepWalker",
+          "log_delay": 5

```

Comment:
hmm, log_delay the wording lead me misunderstood. I was thinking the log will delay to show, that's not what you mean in code. 
Any better wording suggestion since I'm not English speaking guy :)